src/engraving/dom/tuplet.cpp

```diff
@@ -524,6 +524,32 @@ bool Tuplet::cross() const
     return false;
 }
 
+//---------------------------------------------------------
+//   vStaffIdx
+///  Staff index for layout based on the first ChordRest
+//---------------------------------------------------------
+
+staff_idx_t Tuplet::vStaffIdx() const
+{
+    if (elements().empty()) {
+        return mu::nidx;
+    }
+
+    const DurationElement* cr = elements().front();
+    if (!cr) {
+        return mu::nidx;
+    }
+
+    while (cr->isTuplet()) {
+        const Tuplet* t = toTuplet(cr);
+        if (t->elements().empty()) {
+            break;
```

Comment:
I think this could be `return mu::nidx` as well, because if you break from the loop then we will call `Tuplet::vStaffIdx()` recursively for the inner tuplet, which will then return `mu::nidx` anyway.

Also, note that it can theoretically be done using just one while loop, namely by replacing the lines above the while loop with `const DurationElement* cr = this;`. Not sure if that's really nice though.

Reply:
Never mind about my comment above, it's probably not important enough to worry about right now